sql/wsrep_schema.cc

```diff
@@ -1094,6 +1113,9 @@ int Wsrep_schema::remove_fragments(THD* thd,
   }
   else
   {
+    Wsrep_schema_impl::thd_server_status
+      thd_server_status(thd, thd->server_status | SERVER_STATUS_IN_TRANS,
+                        thd->in_multi_stmt_transaction_mode());
```

Comment:
This looks unnecessary. I believe that `thd->in_multi_stmt_transaction_mode()` will always evaluate to true, because:
1) If the thd is local, then thd->in_multi_stmt_transaction() must evaluate to true, because of the outer `if` statement
2) If the thd is applier, then I believe applier always applies transactions in multi stmt mode, even if originally the master was executing it in autocommit mode